hive-metastore/src/test/java/org/apache/iceberg/hive/TestHiveMetastore.java

```diff
@@ -191,6 +193,10 @@ public void reset() throws Exception {
     }
   }
 
+  public Table getTable(String dbName, String tableName) throws TException, InterruptedException {
```

Comment:
Do we want to remove `clientPool()` after this change? It is a dead code now.

Reply:
I think so. Keeping the method gives us future flexibility to run custom metastore client methods without touching the TestHiveMetastore interface, but at the same time, it feels like we're breaching the TestMetastore abstraction by exposing its client pool directly, so it's better to introduce new methods like getTable whenever it becomes necessary.